flink-connectors/flink-connector-hbase-1.4/src/test/java/org/apache/flink/connector/hbase1/HBaseDynamicTableFactoryTest.java

```diff
@@ -183,18 +181,6 @@ public void testTableSinkFactory() {
                 new DataType[] {DECIMAL(10, 3), TIMESTAMP(3), DATE(), TIME()},
                 hbaseSchema.getQualifierDataTypes("f4"));
 
-        Properties properties = new Properties();
-        properties.setProperty("hbase.security.authentication", "kerberos");
-        HBaseOptions expectedHBaseOptions =
-                HBaseOptions.builder()
-                        .setTableName("testHBastTable")
-                        .setZkQuorum("localhost:2181")
-                        .setZkNodeParent("/flink")
-                        .setHbaseProperties(properties)
-                        .build();
-        HBaseOptions actualHBaseOptions = hbaseSink.getHBaseOptions();
-        assertEquals(expectedHBaseOptions, actualHBaseOptions);
```

Comment:
We should add another way to test the configurations. For example : we can get the Hadoop Configruation and convert it into an Map via `Configruation.iterator()`, the we can use `assertEquals` to verify the configs. 

We should also remember to verify table name separately, because it is not in the Hadoop Configruation now.